Kc tracking last response

kev1n80 opened the pull request:
+ Only display the search listings from the latest rpc.
  - Add a TrackingResponse java object in order to track the response of a servlet.
  - Add js functions to handle JSON that respresents a TrackingResponse object.
    + These functions should be used to track the response of a servlet in the front end.
  - Update other functions to incorporate the new TrackingResponse related functions.

zrideaux commented on src/main/java/com/google/sps/servlets/FetchListings.java:
```diff
@@ -110,6 +111,14 @@ public void doGet(HttpServletRequest request, HttpServletResponse response)
 
+    int timeToBack;
+    try {
+      timeToBack = ValidateInput.getParameter(request,"call", 0);
```
tiny nit: add a space before "call"

kev1n80 replied:
Nice catch!

kev1n80 commented:
> Looks good -- lot of cool considerations here (like using the default parameter in `getListings`). I'm curious how you test this kind of thing!

I added logs to see what the `window.lastCall` variable and the call associated with the rpc was. Also, I added logs to see if my `isLatestCall` function was working properly. Finally, I messed with the location, filter, radius, and sort by functionalities on the search page to see if the listings returned came out weird.
